### PR TITLE
Target Laminas

### LINKED ISSUE #15: Any plan to migrate this package to Laminas ?
hello, @weierophinney  I would appreciate it if you can migrate this package. as this package is base of few patterns to me for important projects and uses cases.

### PR DESCRIPTION
**Pull request by weierophinney**
Per the request made in #15, this patch:

- Migrates the package to target Laminas MVC instead of zend-mvc.
- Removes the dependency on zend-mvc-console in favor of the symfony/console-powered `phly-simple-page` vendor binary.

I've also increased minimum requirements as follows:

- laminas-mvc/servicemanager/eventmanager: v3 releases
- PHP: 7.2+

Fixes #15

### CODE REVIEW

**Comment by weierophinney** [comment]
As it is, the minimum version I can support is 7.0, due to my requirement on ocramius/package-versions (7.0 was the earliest version supported by any version of that package).

**Comment by vrkansagara** [comment]
Is it possible to add the list of available cached routes with a file name and it expires time in the console command?

**Comment by weierophinney** [comment]
> Is it possible to add the list of available cached routes with a file name and it expires time in the console command?

That would be a new feature. Put in a request for that after the release, please. :smile:

File: src/Module.php [hunk 1]
```diff
@@ -1,43 +1,38 @@
...
      */
-    public function getConsoleUsage()
+    public function getConfig(): array
     {
-        return [
-            'phlysimplepage cache clear all' => 'Clear caches for all static pages',
-            'phlysimplepage cache clear --page=' => 'Clear caches for a single static page',
-            ['--page', 'Page name as matched via routing'],
-        ];
+        return include __DIR__ . '/config/module.config.php';
```
Comment (vrkansagara) [inline src/Module.php:35]: @weierophinney  `config` directory is one level up. Can you please do the needful. Thanks.

Comment (vrkansagara) [reply src/Module.php:35]: @weierophinney  proposed PR https://github.com/phly/PhlySimplePage/pull/20

**Pull request merged**